src/tsconfig.app.json

```diff
@@ -3,7 +3,7 @@
   "compilerOptions": {
     "outDir": "../out-tsc/app",
     "baseUrl": "./",
-    "module": "es2015",
+    "module": "esnext",
```

Comment:
is it completely necessary?

Reply:
@valorkin  dynamic import won't work without this

Reply:
my mistake